Client/src/Forms/FormSuccess.js

```diff
@@ -1,3 +1,12 @@
+/** [FormSuccess.js]
+* Summary.
+After user user just finished creating profile, the system will show success message.
```

Comment:
* repeated word "user".
* modify to "the system will *display a* success message"

Reply:
done